custom_models/mental_health/models/mental_health_notes.py

```diff
@@ -11,4 +11,102 @@ class MentalHealthNotes(models.Model):
+                                                     ('threat', 'Threat'),
+                                                     ('gesture', 'Gesture'),
+                                                     ('rehearsal', 'Rehearsal'),
+                                                     ('attempt', 'Attempt'),
+                                                     ('plan', 'Plan')],
+                                          default='none',
+                                          help='Is the client suicidal or homicidal?')
+    insight = fields.Selection(string='Insight', required=True, copy=False,
+                               selection=[('poor', 'Poor'),
+                                          ('limited', 'Limited'),
+                                          ('fair', 'Fair'),
+                                          ('good', 'Good')],
+                               default='good',
+                               help='Select the level of insight of the client.')
+    participation = fields.Selection(string='Participation Level', required=True, copy=False,
+                                     selection=[('active', 'Active'),
+                                                ('variable', 'Variable'),
+                                                ('responsive only', 'Responsive only'),
+                                                ('minimal', 'Minimal'),
+                                                ('resistant', 'Resistant')],
+                                     default='active',
+                                     help='Select the participation level of the client.')
+    motivation = fields.Selection(string='Motivation', required=True, copy=False,
+                                  selection=[('low', 'Low'),
+                                             ('moderate', 'Moderate'),
+                                             ('high', 'High')],
+                                  default='high',
+                                  help='Select the level of motivation of the client.')
+    affect = fields.Selection(string='Affect', required=True, copy=False,
+                              selection=[('normal', 'Normal'),
+                                         ('slumped', 'Slumped'),
+                                         ('slowed', 'Slowed'),
+                                         ('confident', 'Confident'),
+                                         ('energetic', 'Energetic'),
+                                         ('nervous', 'Nervous'),
+                                         ('fidgety', 'Fidgety')],
+                              default='normal',
+                              help='Select the affect of the client')
+    symptom_change = fields.Selection(string='Change in Symptom Severity', required=True, copy=False,
+                                      selection=[('resolved', 'Resolved'),
+                                                 ('much improved', 'Much improved'),
```

Comment:
Can we have the naming of the key be standardized to "much_improved" where any space is denoted with a "_"?